original.html

```diff
@@ -1,94 +1,136 @@
-                <a href="#">Contact</a>
-            </nav>
-
-            <img id="logo-img" class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-        </header>
-
-        <section class="cta">
-            <div class="cta-text">
-                <h1>DOM<br> Is<br> Awesome</h1>
-                <button>Get Started</button>
-            </div>
-            <img id="cta-img" src="img/header-img.png" alt="Image of a code snippet.">
-        </section>
-
-        <section class="main-content">
-
-            <div class="top-content">
-                <div class="text-content">
-                    <h4>Features</h4>
-                    <p>Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
-                </div>
-                <div class="text-content">
-                    <h4>About</h4>
-                    <p>About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
-                </div>
-            </div>
-
-            <img class="middle-img" id="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-
-            <div class="bottom-content">
-                <div class="text-content">
-                    <h4>Services</h4>
-                    <p>Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
-                </div>
-                <div class="text-content">
-                    <h4>Product</h4>
-                    <p>Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
-                </div>
-                <div class="text-content">
-                    <h4>Vision</h4>
-                    <p>Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
-                </div>
-            </div>
-
-        </section>
-
-        <section class="contact">
-
-            <h4>Contact</h4>
-
-            <p>123 Way 456 Street<br>
-            Somewhere, USA</p>
-
-            <p>1 (888) 888-8888</p>
-
-            <p>sales@greatidea.io</p>
-
-        </section>
-
-        <footer>
-            <p>Copyright Great Idea! 2018</p>
-        </footer>
-
-    </div><!-- container -->
-</body>
-</html>
\ No newline at end of file
+      <header>
+        <nav>
+          <a href="#">Services</a>
```

Comment:
This looks like you updated the HTML directly...that was EXPRESSLY FORBIDDEN for this project. Let me know if this is wrong - sometimes Github reports some odd things . . . but it looks like you changed the HTML.

Reply:
Thats odd. I definitely didn't make any changes to the HTML. Only thing I remember doing was adding HTML comments as I worked my way down the file so I could quickly spot the area I was working in.